deploy/docker-swarm/clickhouse-setup/docker-compose.yaml

```diff
@@ -111,7 +111,7 @@ services:
       - clickhouse
 
   otel-collector-metrics:
-    image: signoz/signoz-otel-collector:0.55.3
+    image: signoz/signoz-otel-collector:0.63.0
```

Comment:
When is release published for this?

Reply:
@prashant-shahi I don't see a release published for this yet